connect/api/src/main/java/org/apache/kafka/connect/health/ConnectClusterDetails.java

```diff
@@ -27,6 +27,6 @@ public interface ConnectClusterDetails {
      * Get the cluster ID of the Kafka cluster backing this Connect cluster.
      * 
      * @return the cluster ID of the Kafka cluster backing this Connect cluster
-     **/
+     */
     String kafkaClusterId();
 }
```

Comment:
Not really relevant, but this file seems to miss a break line at the end of the file.

Reply:
It think it's just an encoding question between Linux / Window file system -- I guess it was automatically updated when I edited the file.